internal/subscription/validation.go

```diff
@@ -65,6 +65,13 @@ func GetValidatedModelForCreate(ctx context.Context, req *metrov1.Subscription)
 		MaxDeliveryAttempts: req.DeadLetterPolicy.GetMaxDeliveryAttempts(),
 	}
 
+	if req.RetryPolicy != nil {
```

Comment:
damn! big brain fart on my part. Must have missed this while moving code snips around 🤦🏼